apps/hvn/src/app/[locale]/announcements/[slug]/page.tsx

```diff
@@ -0,0 +1,69 @@
+import { draftMode } from 'next/headers'
+import { notFound } from 'next/navigation'
+import { getPayload, TypedLocale } from 'payload'
+
+import { AnnouncementDetail } from '@/components/announcement/AnnouncementDetail'
+import { LOCALES } from '@/i18n/locales'
+import { Announcement } from '@/payload-types'
+import configPromise from '@/payload.config'
+
+type Args = {
+  params: Promise<{
+    locale: TypedLocale
+    slug: string
+  }>
+}
+
+export async function generateStaticParams() {
+  const config = await configPromise
+  const payload = await getPayload({ config })
+
+  const announcements = await payload.find({
```

Comment:
@MustafaTRKYLMZ @myniqx  Does this retrieve announcements in all available languages, or only in the default language?